examples/cms-magnolia/templates/components/List.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { EditableArea } from '@magnolia/react-editor';
+
+export default function List(props) {
```

Comment:
Could we move this example to the App Router?

Reply:
Hey Rob hope you had a good NY's. Currently at Magnolia we don't support the App router because of some context issues in our React Components. I have heard a fix it due to be released and will endevour to update this project once its tested and released.